project/Dependencies.scala

```diff
@@ -12,7 +12,7 @@ object Dependencies {
   - http://doc.akka.io/docs/akka/2.4/scala/http/common/json-support.html#akka-http-spray-json
    */
   lazy val sprayJsonV = "1.3.1"
-  lazy val akkaV = "2.3.15"
+  lazy val akkaV = "2.4.7"
```

Comment:
To fend off any questions asking "why not 2.4.9", it appears that the lightbend reactive platform is built against 2.4.7. It might not matter, in which case i'll later go to 2.4.9